src/main/java/CREResultsRetriever.java

```diff
@@ -63,5 +69,33 @@ private void prepareCREResults() throws IOException {
                 }
             }
         }
+        openFile();
+        addResultsToFile();
+        closeFile();
+    }
+
+    private void openFile() {
+        Date date = new Date();
+        String stringDate = date.toString().replaceAll(":", "-");
```

Comment:
I suppose you don't need this line if you're creating files once per day. Maybe better will be to use Java 8 style to obtain date without time part (LocalDate class).